GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -524,7 +524,10 @@ private void OnNewFileCreated(string relativePath, bool isDirectory)
                         }
                         else
                         {
-                            this.FileSystemCallbacks.OnFolderCreated(relativePath);
+                            if (this.FileSystemCallbacks.OnFolderCreated(relativePath))

```

Comment:
Should we include any comments on why we are calling `OnEnemerateDirectory` callback in response to file creation (when return value is true)?